locales/es.json

```diff
@@ -17,19 +17,28 @@
   },
   "items": {
     "battery_storage_installation": "Instalación de baterías",
+    "ebike": "Bicicleta Eléctrica",
     "efficiency_rebates": "Bonificaciones por eficiencia",
+    "electric_outdoor_equipment": "Equipo eléctrico para exteriores",
     "electric_panel": "Cuadro eléctrico",
     "electric_stove": "Estufa eléctrica/de inducción",
+    "electric_thermal_storage_and_slab": "Almacenamiento de energía térmica y el bloque",
     "electric_vehicle_charger": "Cargador de vehículos eléctricos",
     "electric_wiring": "Cableado eléctrico",
+    "evaporative_cooler": "Enfriador evaporativo",
     "geothermal_heating_installation": "Instalación de calefacción geotérmica",
     "heat_pump_air_conditioner_heater": "Aire acondicionado/calentador con bomba de calor",
     "heat_pump_clothes_dryer": "Secadora de ropa con bomba de calor",
     "heat_pump_water_heater": "Calentador de agua con bomba de calor",
     "new_electric_vehicle": "Nuevo vehículo eléctrico",
+    "non_heat_pump_clothes_dryer": "Secadora de ropa eléctrica",
+    "non_heat_pump_water_heater": "Calentador de agua eléctrico",
+    "other": "Otro",
     "rooftop_solar_installation": "Instalación solar en el techo",
+    "smart_thermostat": "Termostato inteligente",
     "used_electric_vehicle": "Vehículo eléctrico usado",
-    "weatherization": "Climatización"
+    "weatherization": "Climatización",
```

Comment:
This is different from what the embed frontend has. It refers to HVAC as `sistemas de climatización` and weatherization as `impermeabilización`. I'm OK with leaving it for now, since I'm not even sure if we have RA-wide standards for this.